src/cli/init_command.zig

```diff
@@ -291,6 +291,7 @@ pub const InitCommand = struct {
                 };
 
                 try dev_dependencies.data.e_object.putString(alloc, "bun-types", comptime std.fmt.comptimePrint("^{any}", .{version.fmt("")}));
+                try dev_dependencies.data.e_object.putString(alloc, "typescript", "^5.0.0");
```

Comment:
can we make it a peerDependency? installing typescript is like 50 MB

Reply:
That would be unconventional setup. Most TS users will have some version matching `^5.0.0` in the cache already I think. Not doing this is just kinda incorrect. The `bun create` templates are already installing typescript 🤷‍♂️

We can get rid of this once most people are using VSCode versions with ts 5.0+ baked in but right now it would be iffy

Reply:
@Jarred-Sumner ok it's a peer dep now